mirgecom/integrators/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .explicit_rk import *             # noqa: F401, F403
```

Comment:
Are people okay with this? My reasoning here is that all relevant imports through * are determined via `__all__` in the relevant python file. It just makes it so that you don't have to always modify the init when ever someone adds a new timestepper.

Reply:
No idea. 🙂 @inducer

Reply:
No, sorry, I don't like `*` imports much. Here, I suspect it's defensible, but in modules containing code, it removes Flake8's ability to figure out what names are defined. It's noisy, but I'd prefer making the list of imported things explicit.